src/arrayMethodSort.js

```diff
@@ -5,7 +5,32 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let compare = compareFunction;
+
+    if (compare === undefined) {
+      compare = (a, b) => {
+        if (a.toString() > b.toString()) {
+          return 1;
+        } else if (a.toString() < b.toString()) {
+          return -1;
+        }
+
+        return 0;
+      };
+    }
```

Comment:
create separate function outside the `sort2` method and set is as default for `compareFunction` argument